client/src/graphics/voxels/mod.rs

```diff
@@ -142,7 +141,7 @@ impl Voxels {
         for &(node, ref node_transform) in &nodes {
             let node_to_view = local_to_view * node_transform;
             let origin = node_to_view * math::origin();
-            if !frustum_planes.contain(&origin, dodeca::BOUNDING_SPHERE_RADIUS) {
+            if !frustum_planes.contain(&origin, common::dodeca::BOUNDING_SPHERE_RADIUS) {
```

Comment:
Now you've instead introduced extra, unnecessary qualification. Please restore this to the original form, `dodeca::BOUNDING_SPHERE_RADIUS`. In general, avoid modifying code that is not related to the intent of your changes.